Keas.Mvc/Controllers/TagsManagerController.cs

```diff
@@ -38,11 +38,28 @@ public ActionResult Create()
         // POST: Tags/Create
         [HttpPost]
         public async Task<ActionResult> Create( Tag newTag)
-        {
+        {
+            var team = await _context.Teams.FirstAsync(t => t.Slug == Team);
```

Comment:
Unrelated to your PR, but we should probably just throw this in a cache somewhere.  We look up the team all the time and it'd be really easy to cache into a service.